### ISSUE TITLE
Muzzle offsets for the AA gun & Turret disrupted by attacks.

### ISSUE THREAD

**Issue by FrameLimiter** [OpenRA/OpenRA#12898]
If you build a Turret or an AA gun next to some enemies, the enemy's attack creates an issue where the muzzle offsets display incorrectly.

Affects RA & TD, 20161019 & 20170304.
![openra-2017-03-03t160033481z](https://cloud.githubusercontent.com/assets/22008761/23582879/dcd6f2b8-0102-11e7-8673-1879d3913934.png)

**Comment by reaperrr** [comment]
@FrameLimiter Does the muzzle stay there in the upper middle all the time (regardless of facing) or is "only" the offset *relative* to facing wrong?

**Comment by FrameLimiter** [comment]
Yeah the turret/AA gun facing doesn't seem to matter, it can fire in any direction and the offsets are still screwed up.  It seems dependent on the attacks, once there is an interval or break in the attacking fire the muzzle offsets correct themselves.

**Comment by reaperrr** [comment]
Duplicate of #10438.

**Comment by abcdefg30** [comment]
#10438 was fixed but this is still present.
